src/components/TermsAndConditions.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+
+interface Props {
+  hasConfirmed: boolean
+  onConfirmChange: React.ChangeEventHandler<HTMLInputElement>
+}
+
+const TermsAndConditionsPresentation: React.SFC<Props> = props => (
+  <div className="field">
+    <div className="control">
+      <input
+        type="checkbox"
+        className="is-checkradio"
+        id="tcs"
+        name="tcs"
+        checked={props.hasConfirmed}
+        onChange={props.onConfirmChange}
+      />
+      <label htmlFor="tcs">I agree with the</label>
+      <a href="https://kinesis.money" target="_blank">
```

Comment:
I think it would be good to take them to the actual terms page, because it's a bit hard to find on the kinesis money site - eg. `https://kinesis.money/en/terms-of-use/`

Reply:
Yeah this is just temporary until we get the right terms.